kibbeh/src/ui/UserWideButton.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import { SingleUser } from "./UserAvatar";
+import { Button } from "./Button";
+import { BaseUser } from "@dogehouse/kebab";
+
+export interface UserWideButtonInfoProps {
+  user: BaseUser;
+}
+
+const divStyle = {
+  width: "719px",
+  height: "143px",
+  background: "#151A21",
+};
+
+const avatarStyle = {
+  width: "44px",
+  height: "44px",
+  left: "-2px",
+  top: "-3px",
+  marginLeft: "57px",
+  marginBottom: "53px",
+  marginTop: "46px",
+};
+
+const buttonStyle = {
```

Comment:
Please read again the instructions in the issue. You have to use tailwind and all of those margins, tops, height, etc, make no sense. Take the context from where this will be placed and implement it as such.

Adding 100% width and flex should be enough to make this work.

Reply:
sure, will work on it.